### DISCUSSION TITLE
Review

### DISCUSSION THREAD

**Discussion by FennyWilriani** [lab-brussels-1/loruki-group-2#4] [General]
Hey guys, 

@Shewitttt, @alihaidermalik20, @KavehMrh. I'm adding communication-plan.md along with some details and availability. I saw Ali made it as well yet hasn't made a pull request. Lemme know if i should make any changes for this issue 😺

**Comment by alihaidermalik20** [comment]
I have copied our last communication plan with minor changes. So it includes everyone's availability. Should've put it to in progress on project board

**Comment by alihaidermalik20** [comment]
I see you've created a PR with just your times. 
I think it's better if i push my branch and then if there's anything you want to change from our last communication plan, you can do so

**Reply by FennyWilriani** [reply]
Yesss, because I wasn't really sure if everybody will have the same availability, and you also already filling in the details on what to discuss for the meetings which is awesome, so perhaps you can make the PR and merge it first and i'll adjust the minor changes after that.

**Comment by alihaidermalik20** [comment]
Wait, you can see my branch?

**Reply by FennyWilriani** [reply]
I just saw the content of the planning folder on your branch lol .. was it updated?

**Reply by alihaidermalik20** [reply]
Wow. I didn't even add commit and push that branch yet and you can already see it that means

**Reply by FennyWilriani** [reply]
waittt .. do i look it right? now i started to doubting it 😄 .. cause i was seeing your branch (communication-plan) on loruki-group-2 on github.

**Reply by alihaidermalik20** [reply]
Can you send a screenshot? 😆

**Reply by FennyWilriani** [reply]
so that means it was from the last project? 😄

**Reply by FennyWilriani** [reply]
just a sec

**Reply by FennyWilriani** [reply]
<img width="938" alt="ali comm plan" src="https://user-images.githubusercontent.com/97218974/160297738-c2be5369-6573-429f-ab8f-0e03b4a07fa5.png">

**Reply by FennyWilriani** [reply]
that one ..

**Reply by alihaidermalik20** [reply]
I must've committed and pushed the branch before leaving then 😆 i was in a rush cuz Daniel was closing up

**Reply by alihaidermalik20** [reply]
Otherwise, that's sorcery

**Reply by alihaidermalik20** [reply]
It's so automatic now that i don't even remember doing it

**Reply by alihaidermalik20** [reply]
Can you create a PR for me for this branch or only i can do it? Just to know

**Reply by FennyWilriani** [reply]
😄 indeed .. i wasnt even realize you haven't done git push, commit, etc. 
Im not sure tho if can create a PR for that, haven't try it before .. should i try?

**Comment by FennyWilriani** [comment]
Heyyy .. how you guys doing?
@Shewitttt I hope you feel better than yesterday ❤️ .. @KavehMrh hope we heard from you soon 🤞 
@alihaidermalik20 i have read your backlog and almost finish watching the loruki tutorial 📑 
I hope it's alright to you guy if i go ahead and create an html & css issue for loruki content based on backlog that Ali write?
Also i made temporary README file, perhaps tonight we can review all the PR request.
Cheers, F.
